frontend/app/src/components/accounts/ExchangeBalances.vue

```diff
@@ -0,0 +1,272 @@
+        right
+        dark
+        color="primary"
+        to="/settings/api-keys/exchanges"
+      >
+        <v-icon>
+          fa fa-plus
+        </v-icon>
+      </v-btn>
+      <v-row no-gutters class="exchange-balances__content">
+        <v-col cols="12" class="hidden-md-and-up">
+          <v-select
+            v-model="selectedExchange"
+            filled
+            :items="connectedExchanges"
+            hide-details
+            label="Select Exchange"
+            class="exchange-balances__content__select"
+            @change="openExchangeDetails"
+          >
+            <template #selection="{ item }">
+              <div class="exchange-balances__select__image ml-2 mr-6">
+                <v-img
+                  contain
+                  height="42"
+                  width="42"
+                  :title="exchange"
+                  :src="require(`@/assets/images/${item}.png`)"
+                  class="exchange-balances__select__icon"
+                />
+              </div>
+              <div
+                class="exchange-balances__select__item d-flex flex-column my-3"
+              >
+                <span class="exchange-balances__select__item__title text-h6">
+                  {{ item.charAt(0).toUpperCase() + item.slice(1) }}
+                </span>
+                <span
+                  class="exchange-balances__select__item__amount secondary--text text--lighten-5"
+                >
+                  <amount-display
+                    show-currency="symbol"
+                    fiat-currency="USD"
+                    :value="exchangeBalance(item)"
+                  ></amount-display>
+                </span>
+              </div>
+            </template>
+            <template #item="{ item }">
+              <div class="exchange-balances__select__image ml-2 mr-6">
+                <v-img
+                  contain
+                  height="42"
+                  width="42"
+                  :title="exchange"
+                  :src="require(`@/assets/images/${item}.png`)"
+                  class="exchange-balances__select__icon"
+                />
+              </div>
+              <div
+                class="exchange-balances__select__item d-flex flex-column my-3"
+              >
+                <span class="exchange-balances__select__item__title text-h6">
+                  {{ item.charAt(0).toUpperCase() + item.slice(1) }}
+                </span>
+                <span
+                  class="exchange-balances__select__item__amount secondary--text text--lighten-5"
+                >
+                  <amount-display
+                    show-currency="symbol"
+                    fiat-currency="USD"
+                    :value="exchangeBalance(item)"
+                  ></amount-display>
+                </span>
+              </div>
+            </template>
+          </v-select>
+        </v-col>
+        <v-col cols="2" class="hidden-sm-and-down">
+          <v-tabs
```

Comment:
This seems like a prime example of something to extract in a new component

Reply:
We could potentially split that tab/select mode into two different components and use this one to include the distinct components.

Reply:
I would say that it's a presentation+navigational component with two "modes" (wide mode and tall mode), but I think each implementation will probably differ enough that there may not be enough of a shared basis to abstract a common component out of it. Or did you mean to just make a specific sub-component only this page just to make it less crowded?

Reply:
I mean specific sub-components to make it less crowded :)